### DISCUSSION TITLE
I want the canvas to move automatically when the mouse is in the edge area of the canvas.(selectionOnDrag)

### DISCUSSION THREAD

**Discussion by tjxtjxtjx** [xyflow/xyflow#3262] [Q&A] [answered]
After enabling ‘selectionOnDrag’, we can draw a selection on dragging.
I want the canvas to move automatically, when the mouse is in the edge area of the canvas.
Is there any attribute that can achieve this effect? (Like autoPanOnConnect and autoPanOnNodeDrag)

https://github.com/wbkd/react-flow/assets/135824302/77fce695-fece-46c1-a2f3-6f420b3d22cf

**Comment by moklick** [comment, marked as answer]
This is not supported but I like the idea. We will check if we can add it.
